default-private.env

```diff
@@ -0,0 +1,7 @@
+# Private env vars
```

Comment:
If users want to use this file can they copy paste the value as normal? I know with multiline items such as the github private key it may be wonky?

Reply:
@Jdubrick They'll need to ensure the value is surrounded by single quotes and if the value itself has single quotes in it these will need to be manually escaped for now. I think it would be good to create a helper script for properly setting the values just as rhtap-installer had, but I think this would be too out of scope for this PR.